packages/plugins/documentation/admin/src/pages/PluginPage/index.js

```diff
@@ -5,26 +5,76 @@
  *
  */
 
-import React from 'react';
+import React, { useState } from 'react';
 import { useIntl } from 'react-intl';
-import { CheckPermissions, LoadingIndicatorPage } from '@strapi/helper-plugin';
-import { Layout, HeaderLayout, Main, Button } from '@strapi/parts';
-import ShowIcon from '@strapi/icons/Show';
+import {
+  CheckPermissions,
+  ConfirmDialog,
+  LoadingIndicatorPage,
+  stopPropagation,
+} from '@strapi/helper-plugin';
+import {
+  Layout,
+  HeaderLayout,
+  Main,
+  Button,
+  Table,
+  Tr,
+  Thead,
+  Th,
+  Tbody,
+  Text,
+  Td,
+  Row,
+  TableLabel,
+  ContentLayout,
+  IconButton,
+} from '@strapi/parts';
```

Comment:
I suggest you import components one by one. The documentation plugin is lazy loaded and we don't want to import the whoooole DS when we only need a subset of it :).

```js
import { Button } from '@strapi/parts/Button';
```